src/Report/EnabledMembers.php

```diff
@@ -153,6 +151,7 @@ public function parameterFields(): FieldList
      * @param null $_
      * @param Member $record
      * @return string
+     * @deprecated use MemberExtension::getRegisteredMethodNames() instead
```

Comment:
Do we want to signal a version when these methods going to be removed (if at all)?

Reply:
May as well, have updated